src/databricks/labs/ucx/runtime.py

```diff
@@ -202,15 +300,31 @@ def crawl_permissions(cfg: WorkspaceConfig):
 @task(
     "assessment",
     depends_on=[
-        crawl_grants,
-        crawl_permissions,
-        guess_external_locations,
         assess_jobs,
         assess_clusters,
         assess_pipelines,
         assess_azure_service_principals,
         assess_global_init_scripts,
+        crawl_tables,
     ],
+)
+def setup_view(cfg: WorkspaceConfig):
+    """Creates a database view for capturing following summaries as part of the assessment process:
+    - Unsupported DBR version
+    - Unsupported config
+    - DBFS mount used in configuration
+    - Azure service principal credentials used in config
+    - Unsupported storage type (WASBS, ADL) used in table location
+    - Non Delta table
+    - Table scan failure
+    """
+    backend = RuntimeBackend()
+    backend.execute(cfg.replace_inventory_variable(_get_view_definition("$inventory")))
```

Comment:
So, why do we still need an argument for get view definition function? :) it's redundant.